### PR TITLE
Adding asterisks and bold to required fields in sign up

### PR DESCRIPTION
**Pull request by hannahbergam**
I checked how this looks for all four different states:

Student (and parent signing up on behalf of student)
Teachers 

Folks in EU (dgpr)
Folks who are shown CPA

Screenshot 2024-05-21 at 2 25 56 PM
Screenshot 2024-05-21 at 2 35 39 PM
Screenshot 2024-05-21 at 2 37 33 PM
Screenshot 2024-05-21 at 2 25 47 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1904

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1927,6 +1927,23 @@ a.download-video {
...
+    & > * {
+      font-weight: 600;
+    }
+
+    &::after {
+      content: "*";
+      display: inline;
+    }
+  }
+
```
Comment (kelbyhawn) [inline dashboard/app/assets/stylesheets/application.scss:1946]: Try this to clean up the css a bit!

```suggestion
  .required, 
  .required > * {
    font-weight: 600;

    &.not-bold {
      font-weight: normal;
    }

    &::after {
      content: "*";
      display: inline;
    }
  }

```

**Review by kelbyhawn — APPROVED** [review]
Looks great!

**Review by kelbyhawn — APPROVED** [review]

**Review by bethanyaconnor — APPROVED** [review]
I am way too excited about this tbh

**Comment by kelbyhawn** [comment]
> I am way too excited about this tbh

Me too!!

**Review by carl-codeorg — COMMENTED** [review]
For LTI-based signups, we might not always get an email address for teachers, so we show them the email field on this page. Can you bold that field as well? It's normally auto-populated, but still should be required:
Screenshot 2024-05-22 at 10 32 29 AM

**Comment by hannahbergam** [comment]
> For LTI-based signups, we might not always get an email address for teachers, so we show them the email field on this page. Can you bold that field as well? It's normally auto-populated, but still should be required: Screenshot 2024-05-22 at 10 32 29 AM

Thanks for calling out this case! I wasn't sure how to hack this page to see the field locally, but fixed what I think is the correct label. Are you able to pull the latest from the branch to verify?

**Review by carl-codeorg — APPROVED** [review]
LGTM!

Screenshot 2024-05-22 at 10 57 47 AM

**Pull request merged**